airbyte-cdk/bulk/core/load/src/test/kotlin/io/airbyte/cdk/load/data/TimeStringToIntegerTest.kt

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.cdk.load.data
+
+import io.airbyte.cdk.load.message.DestinationRecord
+import io.airbyte.cdk.load.test.util.Root
+import io.airbyte.cdk.load.test.util.SchemaRecordBuilder
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+
+class TimeStringToIntegerTest {
+
+    @Test
+    fun testMapDate() {
+        val mapper = TimeStringToInteger(DestinationRecord.Meta())
+        listOf(
+                "2021-1-1" to 18628,
+                "2021-01-01" to 18628,
+                "2021/01/02" to 18629,
+                "2021.01.03" to 18630,
+                "2021 Jan 04" to 18631,
+                "2021-1-1 BC" to -1457318
```

Comment:
sounds like we currently support all of these, but... it would be cool not to :(

(also I'm just assuming these numbers are correct, not actually verifying :P )

Reply:
oh derp you're deleting them in https://github.com/airbytehq/airbyte/pull/47354/

Reply:
Another rebase weirdness. That's me re-deleting that integral that popped back up.

These tests seem a little overkill, but I guess there's cases where old dates would make sense. (A database of ancient artifacts?)

The numbers are copied from the old tests in the json-avro-converter repo, some of which I added and verified when I was doing S3. For this I'm just going for parity.